src/com/dabomstew/pkrandom/newgui/Bundle.properties

```diff
@@ -583,4 +583,18 @@ GUI.tmFollowEvolutionsCheckBox.toolTipText=<html>When this is selected and TM co
 GUI.mtFollowEvolutionsCheckBox.toolTipText=<html>When this is selected and move tutor compatibility is randomized, evolutions of a Pokemon will inherit that Pokemon's move tutor compatibilities.<br />Additionally, when a Pokemon evolves, each move will have:<br /><b>Random (prefer same type)</b>: A 90% chance of becoming learnable if the Pokemon just gained its type through evolution, a 10% chance otherwise.<br/><b>Random (completely)</b>: A 25% chance of becoming learnable regardless of type.
 GUI.stpFixMusicAllCheckBox.text=Fix Music
-GUI.stpFixMusicAllCheckBox.toolTipText=<html>Fixes the music for all Static Pokemon encounters so that even when randomized, encounters that should have special music will play the correct song.<br />Note that in Gen 4/5, if special music is assigned to a Pokemon then it will <b>always</b> play when battling that Pokemon, even if you just find it normally in the wild.
\ No newline at end of file
+GUI.stpFixMusicAllCheckBox.toolTipText=<html>Fixes the music for all Static Pokemon encounters so that even when randomized, encounters that should have special music will play the correct song.<br />Note that in Gen 4/5, if special music is assigned to a Pokemon then it will <b>always</b> play when battling that Pokemon, even if you just find it normally in the wild.
+GUI.tpHeldItemsLabel.text=Add Held Items to...
+GUI.tpBossTrainersItemsCheckBox.text=Boss Trainers
+GUI.tpBossTrainersItemsCheckBox.toolTipText=<html>Check this to add or replace held items for the Pokemon of "Boss" Trainers.<br />Boss Trainers include Gym Leaders, Kahunas, Team Leaders, Elite Four, and Champions.
+GUI.tpImportantTrainersItemsCheckBox.text=Important Trainers
+GUI.tpImportantTrainersItemsCheckBox.toolTipText=<html>Check this to add or replace held items for the Pokemon of "Important" Trainers.<br />Important Trainers include Rivals/Friends, Team Admins, and other important story battles (such as Colress in BW2 and Sycamore in XY).
+GUI.tpRegularTrainersItemsCheckBox.text=Regular Trainers
+GUI.tpRegularTrainersItemsCheckBox.toolTipText=<html>Check this to add or replace held items for the Pokemon of regular Trainers.<br />Regular Trainers include all trainers that don't qualify as "Boss" or "Important" Trainers.
+GUI.tpConsumableItemsOnlyCheckBox.text=Consumable Only
+GUI.tpConsumableItemsOnlyCheckBox.tooltip=<html>Items will be chosen randomly from all single-use held items that a Pokemon can eat/use on its own.<br />This is mostly berries, with a few other items like Focus Sash and White Herb.</html>
+GUI.tpSensibleItemsCheckBox.text=Sensible Items
+GUI.tpSensibleItemsCheckBox.tooltip=<html><b>Not implemented yet.</b><br />Items will be chosen randomly from a subset of the held items that "make sense" for a given Pokemon. For example, Silk Scarf won't be an option for a given Pokemon unless it knows at least one normal move. Another example is that a Yache berry will only be an option on Pokemon that are weak or double weak to ice.</html>
```

Comment:
I suggest implementing line breaks for this and the Highest Level Only tooltip, since they look much longer than any other tooltip. There's no hard or fast rule for how long a line should be so just experiment; we tend to put them between sentences.

Reply:
Done.